### ISSUE TITLE
Incompatible API version with plugin. Plugin version: 4, Client versions: [5]

### ISSUE THREAD

**Issue by mshirley** [squat/terraform-provider-vultr#72]
it seems like the latest version of the plugin isn't compatible with the latest version of terraform.  i'll downgrade terraform to a stable version and try again.

```
$ terraform --version
Terraform v0.12.0-dev
+ provider.vultr (unversioned)
```

```
$ terraform init

Initializing the backend...

Initializing provider plugins...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
```

```
$ terraform plan

Error: Failed to instantiate provider "vultr" to obtain schema: Incompatible API version with plugin. Plugin version: 4, Client versions: [5]
```

**Comment by squat** [comment]
Thanks for the heads up! I have not been testing with 0.12.0. I’ll take a look.

**Comment by mshirley** [comment]
confirmed that downgrading to v0.11.13 fixes the issue.

```
$ terraform-v0.11.13 plan
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

data.vultr_region.new_jersey: Refreshing state...
data.vultr_plan.plan_2cpu_4gb: Refreshing state...
data.vultr_os.ubuntu_18-04: Refreshing state...

------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create
<snip>
```

**Comment by jdmulloy** [comment]
All that's needed is upgrading the terraform library. I've submitted a PR to do this upgrade. It works on my system, but I only use the DNS and firewall rules functionally.

squat/terraform-provider-vultr#74

**Comment by squat** [comment]
fixed by #76. Please report back if you still see this issue, @mshirley
